web-common/src/features/dashboards/filters/dimension-filters/dimensionFilterValues.ts

```diff
@@ -37,15 +38,37 @@ export function useDimensionSearch(
   );
 
   return derived(queries, ($queries) => {
+    const someQueryFetching = $queries.some((q) => q.isFetching);
```

Comment:
Typically we should use `isLoading` not `isFetching`

Reply:
Hmmm my understanding was `isLoading` is true only on the 1st fetch. But i guess it is for the 1st fetch of the query object. Since we create a new one on every searchText it will be true for showing the spinner.

Reply:
But if for whatever reason there a refetch the data would go from blank to showing values without a spinner. So I think it is fine to use `isFetching` here.